docs/cli/graphql-transformer/directives.md

```diff
@@ -1912,6 +1912,8 @@ The `@searchable` directive handles streaming the data of an `@model` object typ
 
 > **Note**: `@searchable` is not compatible with DataStore but you can use it with the API category. 
 
+> **Note**: `@searchable` is not compatible with  t2.micro instance as it only works with version 1.5 and 2.3 and Amplify CLI support instances with version >= 6.x.
```

Comment:
what's the version number we're talking out here? Elasticsearch verision?

Reply:
yes, Its the Elasticsearch version

Reply:
@akshbhu @kaustavghosh06 - just updated the copy here to be more explicit. Can you double check?